apps/client/src/features/rundown/Rundown.tsx

```diff
@@ -133,7 +143,16 @@ export default function Rundown(props: RundownProps) {
         }
       }
     },
-    [cursor, entries, insertAtCursor, moveCursorTo],
+    [cursor, entries, insertAtCursor, setEditMode],
+  );
+
+  const handleKeyUp = useCallback(
```

Comment:
could we instead check if the shift or control where pressed at the keydown event?

Reply:
Can you please explain this a bit more? Isn't that happening on line 91/97?